### PR TITLE
[12.0] [MIG] pos_default_quantity

### PR DESCRIPTION
**Pull request by vvrossem**
### [Task](https://gestion.coopiteasy.be/web#id=3153&view_type=form&model=project.task&action=479)

This module was originally in the [`coopiteasy/pos`](https://github.com/coopiteasy/pos/tree/9.0-pos-default-quantity) repository on the `9.0-pos-default-quantity` branch

A decision was taken to move it in `coopiteasy/addons` repository.

### CODE REVIEW

File: pos_default_quantity/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Coop IT Easy SCRLfs
+# 	    Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Set Default Product Quantity in POS",
+    "version": "12.0.1.0.0",
+    "author": "Coop IT Easy SCRLfs, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/pos",
```
Comment (houssine78) [inline pos_default_quantity/__manifest__.py]: c'est notre module il n'est pas sur l'OCA du coup je propose de mettre https://coopiteasy.be

File: pos_default_quantity/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Coop IT Easy SCRLfs
+# 	    Robin Keunen <robin@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Set Default Product Quantity in POS",
+    "version": "12.0.1.0.0",
+    "author": "Coop IT Easy SCRLfs, Odoo Community Association (OCA)",
```
Comment (houssine78) [inline pos_default_quantity/__manifest__.py]: pas de l'OCA il me semble

File: pos_default_quantity/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Coop IT Easy SCRLfs
+# 	    Robin Keunen <robin@coopiteasy.be>
```
Comment (houssine78) [inline pos_default_quantity/__manifest__.py:2]: tu peux ajouter ton nom ici maintenant

File: pos_default_quantity/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,2 @@
+* Robin Keunen <robin.keunen@coopiteasy.be>
+
```
Comment (houssine78) [inline pos_default_quantity/readme/CONTRIBUTORS.rst]: ajoute ton nom ici

File: pos_default_quantity/views/product_view.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```
Comment (houssine78) [inline pos_default_quantity/views/product_view.xml]: should be odoo tag on twelve

File: pos_default_quantity/views/product_view.xml [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    <data>
```
Comment (houssine78) [inline pos_default_quantity/views/product_view.xml]: you can remove data tag if noupdate is not set

File: pos_default_quantity/views/product_view.xml [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (houssine78) [inline pos_default_quantity/views/product_view.xml:1]: name of the field should not be uom_category_view.xml instead ?

File: pos_default_quantity/README.rst [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/pos <https://github.com/OCA/pos/tree/9.0-pos-default-quantity/pos_default_quantity>`_ project on GitHub.
```
Comment (houssine78) [inline pos_default_quantity/README.rst]: it need correction

File: pos_default_quantity/README.rst [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+Authors
+~~~~~~~
+
+* Coop IT Easy SCRLfs
+
+Contributors
+~~~~~~~~~~~~
+
+* Robin Keunen <robin.keunen@coopiteasy.be>
+
```
Comment (houssine78) [inline pos_default_quantity/README.rst]: add you name

**Review by houssine78 — CHANGES_REQUESTED** [review]
small modification requested

File: pos_default_quantity/__manifest__.py (hunk 1 above)
Comment (vvrossem) [reply pos_default_quantity/__manifest__.py]: website changed!

File: pos_default_quantity/__manifest__.py (hunk 2 above)
Comment (vvrossem) [reply pos_default_quantity/__manifest__.py]: OCA removed!

File: pos_default_quantity/__manifest__.py (hunk 3 above)
Comment (vvrossem) [reply pos_default_quantity/__manifest__.py:2]: :2nd_place_medal:

File: pos_default_quantity/readme/CONTRIBUTORS.rst (hunk 4 above)
Comment (vvrossem) [reply pos_default_quantity/readme/CONTRIBUTORS.rst]: :2nd_place_medal:

File: pos_default_quantity/views/product_view.xml (hunk 5 above)
Comment (vvrossem) [reply pos_default_quantity/views/product_view.xml]: shame on me !

File: pos_default_quantity/views/product_view.xml (hunk 6 above)
Comment (vvrossem) [reply pos_default_quantity/views/product_view.xml]: shame on me (bis)

File: pos_default_quantity/views/product_view.xml (hunk 7 above)
Comment (vvrossem) [reply pos_default_quantity/views/product_view.xml:1]: it is `id="product_uom_categ_form_view"` and file is named `product_view.xml` in [12.0](https://github.com/odoo/odoo/blob/12.0/addons/point_of_sale/views/product_view.xml#L87)

File: pos_default_quantity/README.rst (hunk 8 above)
Comment (vvrossem) [reply pos_default_quantity/README.rst]: automatically corrected after executing ` oca-gen-addon-readme --org-name="coopiteasy" --repo-name=addons --branch=12.0 --addon-dir=pos_default_quantity`

File: pos_default_quantity/README.rst (hunk 9 above)
Comment (vvrossem) [reply pos_default_quantity/README.rst]: corrected with `oca-gen-addon-readme`

**Review by remytms — APPROVED** [review]

**Pull request merged**